docs/reference/module-overview.md

```diff
@@ -0,0 +1,291 @@
+# Anatomy of a module
+
+Module configuration objects may use the following configuration properties.
+
+## Configuration settings
+
+| Setting name | Value type | [Cascades](#cascading-settings) | Description | Module types |
+| ------- | ------- | ------- | ------- | :-----: |
+| [`extend`](#extend) | String | No | Identify the base class module | All |
+| [`improve`](#improve) | String | No | Identify a module to enhance | All |
+| [`options`](#options) | Object | No | Configure module options | All |
+| [`instantiate`](#instantiate) | Boolean | No | Prevent a module from being fully instantiated | All |
+| [`fields`](#fields) | Object | Yes | Configure doc type fields | Doc, Widget |
+| [`filters`](#filters) | Object | Yes | Configure piece type filters | Piece |
+| [`columns`](#columns) | Object | Yes | Configure piece type manager columns | Piece |
+
+### `extend`
+
+Identify a module to use as a base class for your custom module. "Extending" a module this way makes all of the functionality and configuration of the base class available in the new module, other than its `name` property.
```

Comment:
"custom module" -- not going to stop this from being 'finished', but flagging this again. Custom implies aberration, and everything....is a module. So I think this lacks a lot of clarity. We're not using any real, relevant descriptors here, imo

Reply:
I think custom implies "your own." But it could just say "a base class for this module."